zksync_sdk/ethereum_signer/web3.py

```diff
@@ -28,7 +28,13 @@ class TxEthValidator(TxEthValidatorInterface):
 
     def is_valid_signature(self, tx: Order) -> bool:
```

Comment:
Why need a separate class for this? The signer address can be derived from `Order` object